BranchMetrics/ios-branch-deep-linking-attribution
# Sdk 1755 remove old apple search ads ap is

## Reference
SDK-1755 -- iOS - remove old Apple Search Ads APIs
https://branch.atlassian.net/browse/SDK-1755

## Summary
Removed all code related to older Apple Search Ads framework -iAD.
Files related to classes BNCAppleAdClient and BNCAppleSearchAds are removed from project.
Following public APIs are deleted 
- (void)delayInitToCheckForSearchAds
- (void)useLongerWaitForAppleSearchAds 
- (void)ignoreAppleSearchAdsTestData
- (void)checkAppleSearchAdsAttribution 

This will break linking in Apps which are using these APIs or classes.
Preferences, Constants and Unit tests related to above code is also removed.

## Motivation
Removed deprecated APIs.

## Type Of Change


## Testing Instructions
Check logs of installs / open requests to see if there is not any field "search_ad_encoded".









cc @BranchMetrics/saas-sdk-devs for visibility.
